tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/GitHubLabelsTool/GitHubLabelsTool.cs

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.ComponentModel;
+using Azure.Sdk.Tools.Cli.Services;
+using Azure.Sdk.Tools.Cli.Contract;
+using ModelContextProtocol.Server;
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Models.Responses;
+using Azure.Sdk.Tools.Cli.Helpers;
+using System.Text;
+using CsvHelper;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+
+
+namespace Azure.Sdk.Tools.Cli.Tools
+{
+
+    [McpServerToolType, Description("Tools for working with GitHub service labels from the Azure SDK Tools common-labels.csv")]
+    public class GitHubLabelsTool(
+        ILogger<GitHubLabelsTool> logger,
+        IOutputService output,
+        ILabelHelper labelHelper,
+        IGitHubService githubService
+    ) : MCPTool
+    {
+        private const string serviceLabelColorCode = "e99695"; // color code for service labels in common-labels.csv
```

Comment:
Let's define this in `Configuration/Constants.cs` since it's used across 3 different classes/tests.